frameworks/hyperoptsklearn/exec.py

```diff
@@ -3,12 +3,15 @@
 import math
 import os
 import signal
+import sys
 import tempfile as tmp
 
 os.environ['JOBLIB_TEMP_FOLDER'] = tmp.gettempdir()
-os.environ['OMP_NUM_THREADS'] = '1'
-os.environ['OPENBLAS_NUM_THREADS'] = '1'
-os.environ['MKL_NUM_THREADS'] = '1'
+if sys.platform == 'darwin':
+    os.environ['OMP_NUM_THREADS'] = '1'
+    os.environ['OPENBLAS_NUM_THREADS'] = '1'
+    os.environ['MKL_NUM_THREADS'] = '1'
+
```

Comment:
@PGijsbers hyperoptsklearn is not included in https://github.com/openml/automlbenchmark/pull/235 and I think it's dead, isn't it?

Reply:
I'm not sure. There doesn't seem to be any active development anymore and it doesn't completely conform to guidelines (e.g. no time-based limit). I'm also not sure how relevant the package still is (it's seems great at what it does, but I haven't seen an AutoML paper use it their comparison in a while).